qa/suites/fs/upgrade/featureful_client/old_client/tasks/2-upgrade.yaml

```diff
@@ -20,9 +20,7 @@ tasks:
 - mds_pre_upgrade:
 - print: "**** done mds pre-upgrade sequence"
 - install.upgrade:
-    mon.a:
```

Comment:
I would add a note (comment) here mentioning the same. And maybe just keep mon.a :)

Reply:
Sure!

It means to upgrade the `mds` stuff:

`print: "**** done mds pre-upgrade sequence"`

I just switched to mds to avoid confusing here. I am okay to change it back!

Reply:
I mean comment in the yaml itself.

Reply:
done! Thanks.